packages/rest/src/types.ts

```diff
@@ -77,6 +78,14 @@ export type LogError = (
 ) => void;
 
 // tslint:disable:no-any
+
+/**
+ * Options for request body parsing
+ */
+export type RequestBodyParserOptions = OptionsJson &
```

Comment:
Is it a good idea to mix all content-type-specific options at the top level?

What if a user wants to configure different limits for JSON and Text payloads? Is it a valid requirement to support?